app/staging/domain/team.js

```diff
@@ -0,0 +1,11 @@
+class Team {
+  constructor (id, lduId, description, effectiveFrom, effectiveTo) {
+    this.id = id
+    this.lduId = lduId
+    this.description = description
+    this.effectiveFrom = effectiveFrom
+    this.effectiveTo = effectiveTo
```

Comment:
In these domain objects, we're likely going to want to start having `isValid` methods to ensure object integrity. When we map these they will act like bastions ensuring the quality of the data that enters the system so I think we should be pretty strict. In order to do this we'll need to know the expected date format etc.